runtime/testy/studentAPI.py

```diff
@@ -78,6 +78,9 @@ def __init__(self, toManager, fromManager):
     super().__init__(toManager, fromManager)
     self._createSensorMapping()
     self._coroutines_running = set()
+    self.hibikeEnumerateDevices()
+    # Wait for hibike to respond
+    time.sleep(0.5)
```

Comment:
Nothing that does not require a decidedly non-trivial amount of additional runtime infrastructure changes. 
This is not actually needed i don't think, because hibike does enumeration already but... 
this is a just in case thing.